packages/libp2p/src/circuit-relay/transport/index.ts

```diff
@@ -296,7 +296,7 @@ class CircuitRelayTransport implements Transport {
   /**
    * Create a listener
    */
-  createListener (options: CreateListenerOptions): Listener {
+  createListener (): Listener {
```

Comment:
Is this change needed?

Reply:
Doesn't look like it, no.

Reply:
The argument is not being used so I am not sure why it's being passed.